sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -2472,7 +2520,7 @@ object Radar {
                     return
                 }
 
-                logConversion(name, data, location, events, user, callback)
+                sendLogConversionRequest(name, data, location, events, user, callback)
```

Comment:
`location, events, user,` are all unnecessary for the purposes of `POST /v1/events`

Reply:
still remains